tests/test_theme.py

```diff
@@ -45,7 +45,7 @@ def test_depth_peeling_eq(default_theme):
 @pytest.mark.parametrize(
     'parm',
     [
-        ('color', (0.1, 0.1, 0.1)),
+        ('color', (0.2, 0.2, 0.2)),
```

Comment:
These were originally necessary because `pyvista.Color((0.1, 0.1, 0.1)).f_rgb` is not equal to `(0.1, 0.1, 0.1)` because of the internal integer representation. However, now that we have an `__eq__` method and the internal color members are all converted to be `Color` instances instead of float sequences, I think most of these can actually be changed back to their old values (I will verify this).

Reply:
I was indeed able to revert these back in the last commit.